Week-1/styles.css

```diff
@@ -0,0 +1,115 @@
+
+body {
+  display: flex;
+  flex-direction: column;
+  background-color: white;
+  margin: 0;
+  font-family: Lato, sans-serif;
+  color: black;
+}
+
+main {
+  flex: 1 0 auto;
+  display: flex;
+}
+
+aside {
+  flex: 0 0 40px;
+  display: flex;
+  flex-direction: column;
+  justify-content: space-around;
+  align-items: center;
+  background-color: #f2f2f2;
```

Comment:
Lets start using CSS `variables` for defining colors. 

https://developer.mozilla.org/en-US/docs/Web/CSS/Using_CSS_custom_properties

Reply:
Yes added  CSS  variables for color and background color .